src/test/unit/extension/conftest.py

```diff
@@ -31,6 +32,24 @@ def mock_linux_files():
         yield
 
 
+def clean_folder(folder):
+    for filename in os.listdir(folder):
+        file_path = os.path.join(folder, filename)
+        try:
+            if os.path.isfile(file_path) or os.path.islink(file_path):
+                os.unlink(file_path)
+            elif os.path.isdir(file_path):
+                shutil.rmtree(file_path)
+        except Exception as e:
+            print('Failed to delete %s. Reason: %s' % (file_path, e))
+
+
+@pytest.fixture(autouse=True)
+def extension_clean():
+    clean_folder('/tmp/lumigo-spans')
+    yield
+    clean_folder('/tmp/lumigo-spans')
```

Comment:
You can simplify it:  🙂 
```suggestion
    if os.path.exists('/tmp/lumigo-spans'):
        shutil.rmtree('/tmp/lumigo-spans')
    yield
```